assignment-2/out/production/assignment-2/README.md

```diff
@@ -0,0 +1,19 @@
+#Student Management Application
```

Comment:
You should move this file from `assignment-2/out/production/assignment-2` to `assignment-2` folder.